fashion_mnist/INSTALL.md

```diff
@@ -0,0 +1,55 @@
+# Creating VM for Fashion MNIST Tutorial
+
```

Comment:
These instructions describe how to install a VM for the [Fashion MNIST Tutorial](README.md).